libraries/AP_OpticalFlow/OpticalFlow.cpp

```diff
@@ -128,6 +129,9 @@ void OpticalFlow::init(uint32_t log_bit)
     case OpticalFlowType::CXOF:
         backend = AP_OpticalFlow_CXOF::detect(*this);
         break;
+    case OpticalFlowType::UPFLOW:
```

Comment:
maybe put it a little lower in the list.  it seems all the existing items in are numerical order so let's keep it like that.